xbmc/utils/LabelFormatter.cpp

```diff
@@ -318,6 +318,8 @@ std::string CLabelFormatter::GetMaskContent(const CMaskString &mask, const CFile
   case 'a': // Date Added
     if (movie && movie->m_dateAdded.IsValid())
       value = movie->m_dateAdded.GetAsLocalizedDate();
+    if (music)
+      value = music->GetDateAdded().GetAsLocalizedDate();
```

Comment:
Tried that again, works now. Must have failed due to the XX/XX/1601 date it returned when I worked on that part.